src/backend/aspen/api/views/tests/test_auth.py

```diff
@@ -233,27 +233,27 @@ async def test_sync_complicated_roles(
     await start_new_transaction(async_session)
 
     # NOTE - DO NOT USE groups[0] in our test for fear of the SA identity map dragon!!!
```

Comment:
omg 😂

Reply:
seriously. hours. hours of wasted time. ☠️